src/index.html

```diff
@@ -7,7 +7,55 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="rectangle_with_stars">
+
+    <block class="stars--0 block-stars">
```

Comment:
just stars, also use div

```suggestion
    <div class="stars--0 stars">
```